deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -11,6 +11,7 @@
 </ul>
 
 {% if pinterest %}
+    {% include "deploys/warning_no_nimbus_project_no_deploy.tmpl" %}
```

Comment:
can you check the env's project first and only display the warning for the env that does not have an owner?